pkg/sql/sem/tree/pretty.go

```diff
@@ -1116,6 +1124,35 @@ func (node *CreateTable) doc(p *PrettyCfg) pretty.Doc {
 	return p.nestUnder(title, pretty.Group(pretty.Stack(clauses...)))
 }
 
+func (p *PrettyCfg) createDefs(defs TableDefs) pretty.Doc {
```

Comment:
Can this be `func (node TableDefs) doc (p *PrettyCfg) pretty.Doc` instead? Otherwise users have to know about this function instead of being able to delegate to the p.Doc func like normal. For example IMPORT also could benefit from this.

Reply:
Good idea. Done.